pages/admin/index.tsx

```diff
@@ -272,7 +272,12 @@ export default function Requests() {
             </Flex>
             <Table columns={COLUMNS} data={DATA} />
             <Flex justifyContent="flex-end">
-              <Pagination pageSize={20} totalCount={150} />
+              <Pagination /* eslint-disable @typescript-eslint/no-empty-function */
+                pageNumber={0}
+                onPageChange={() => {}}
```

Comment:
cc @angeladietz @jennifertsai we moved the state management for the page number to be outside of the `Pagination` component, so you will need to define this state yourself and pass it into `Pagination`.